functionary/ui/views/function.py

```diff
@@ -114,14 +112,37 @@ def execute(request: HttpRequest) -> HttpResponse:
                 ),
             )
         except S3Error:
+            task = None
             status_code = 503
             form.add_error(
                 None,
                 (
-                    "Unable to upload file. Please try again. "
+                    "Unable to upload file, please try again. "
                     "If the problem persists, contact your system administrator."
                 ),
             )
+        except Exception:
```

Comment:
Let's just let this exception bubble up and not handle it.  If we get here it's because we screwed something up.